packages/discovery-provider/src/tasks/entity_manager/entities/playlist.py

```diff
@@ -276,6 +330,8 @@ def update_playlist(params: ManageEntityParameters):
 
     update_playlist_routes_table(params, playlist_record, False)
 
+    update_playlist_tracks_relations(params, playlist_record)
```

Comment:
just checking why we have this twice? is this playlist vs album?

Reply:
top one is on create playlist action, 2nd one is on update playlist action.
i believe playlists and albums are all considered collections.

Reply:
in discovery, albums are in the playlists table. kinda nit but maybe it should just be called playlist_tracks because it's indexing off of playlists